node-api/backend/types.go

```diff
@@ -67,7 +67,10 @@ type BlockStore[BeaconBlockT any] interface {
 // DepositStore defines the interface for deposit storage.
 type DepositStore interface {
 	// GetDepositsByIndex returns `numView` expected deposits.
-	GetDepositsByIndex(startIndex uint64, numView uint64) ([]*ctypes.Deposit, error)
+	GetDepositsByIndex(
+		startIndex uint64,
+		numView uint64,
+	) ([]*ctypes.Deposit, error)
```

Comment:
Do you have your editor still setup to format all lines over 80 chars? It happens also in several other places